built_value_generator/lib/src/serializer_source_field.dart

```diff
@@ -65,6 +65,11 @@ abstract class SerializerSourceField
         compare: annotation.getField('compare').toBoolValue(),
         serialize: annotation.getField('serialize').toBoolValue(),
         wireName: annotation.getField('wireName').toStringValue(),
+        oldWireNames: annotation
```

Comment:
Since this a new field the generator will need to be robust to it being missing; in that case `annotation.getField('oldWireNames')` will return null. See "settings" in value_source_class.dart, although this is a little harder since it's a list.